include/flatbuffers/base.h

```diff
@@ -16,6 +16,11 @@
   #define new DEBUG_NEW
 #endif
 
+#if defined(_MSC_VER) && defined(_DEBUG)
+// see: https://devblogs.microsoft.com/cppblog/why-am-i-getting-these-_scl_secure_no_warnings-messages/
+#define _SCL_SECURE_NO_WARNINGS 1
```

Comment:
I'm not sure that this definition in the header file will be ok.
It is possible to declare this inside cpp files (e.g. utils.cpp).
But in any library header file, it will be like a virus that can hide problems not connected with flatbuffers at all.
Why this definition is needed?

Reply:
It was for these:

`std::transform(v, v + len, StartVectorOfStructs<T>(len), pack_func);`

as the `StartVectorofStructs<T>` returns just a raw pointer, so it is complaining about bounds checks.

Yeah, i can move it to the .cpp file, but let me see if I can just add an ignore warnings to the individual methods.

Reply:
I just changed them to simple for loops to avoid the whole issue.